XMTP.podspec

```
@@ -44,5 +44,5 @@ Pod::Spec.new do |spec|
   spec.dependency "web3.swift"
   spec.dependency "GzipSwift"
   spec.dependency "Connect-Swift", "= 0.3.0"
-  spec.dependency 'XMTPRust', '= 0.3.7-beta0'
+  spec.dependency 'LibXMTP', '= 0.3.7-beta0'
```

```suggestion
  spec.dependency 'LibXMTP', '= 0.0.1-timeout0'
```

Set to the verision from [LibXMTP podspec](https://github.com/xmtp/libxmtp-swift/blob/main/LibXMTP.podspec#L3)

---

Thats an odd name. 😄 probably something we want to fix eventually.